<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridWidget</class>
 <widget class="QWidget" name="GridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>76</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>343</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLabel" name="yeTextLabel">
     <property name="text">
      <string>y&lt;sub&gt;e&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="xbTextLabel">
     <property name="text">
      <string>x&lt;sub&gt;b&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QDoubleSpinBox" name="dxDoubleSpinBox">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="xbDoubleSpinBox">
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>-4.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="ybDoubleSpinBox">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>-4.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="xeTextLabel">
     <property name="text">
      <string>x&lt;sub&gt;e&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="dyTextLabel">
     <property name="text">
      <string>dy</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QDoubleSpinBox" name="dyDoubleSpinBox">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QDoubleSpinBox" name="xeDoubleSpinBox">
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>4.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="dxTextLabel">
     <property name="text">
      <string>dx</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ybTextLabel">
     <property name="text">
      <string>y&lt;sub&gt;b&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QDoubleSpinBox" name="yeDoubleSpinBox">
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>4.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
